Mod1/Labs/Week1/ConditionalLogic.md

```diff
@@ -7,20 +7,20 @@ Plan for at least 2 whole group checkins - after giving some time to pseudo-code
 
 <!-- I definitley agree that labs should be as collaborative as possible - I think at some point our instructor notes should be more explicit such as "Before this lab, create groups of 2-3 students and post groups in cohort Slack channel at the start of the lab." Almost making it like a checklist so it is easy for future instructors to prep. -->
 
-## Practice
+<!-- I'm in favor of creating random breakout room groups for students at the start of the lab. Is there a reason you prefer to post the groups in slack instead of creating breakout rooms with the groups? -->
```

Comment:
Yes, generally I am also in favor of random groups.  But, Eric and I were discussing the benefits of maybe having a core group of partners that students work with during the mod.  The idea being that they might be more comfortable if they can build some relationships while working on code.  This could also be my own anxiety of being dumped into a random room 😆

Reply:
I think this will ultimately be something that evolves as the instructional staff changes, and as we get to know this student population better